src/App.js

```diff
@@ -1,10 +1,21 @@
 import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import './App.css';
+import Header from './components/public/Header';
```

Comment:
Can you rename the public folder to layouts inside the components folder? It's more descriptive that way.